package.json

```diff
@@ -12,12 +12,15 @@
     "@next-auth/supabase-adapter": "^0.2.1",
     "@supabase/supabase-js": "^2.20.0",
     "autoprefixer": "10.4.14",
+    "formik": "^2.2.9",
```

Comment:
**General Comment:**
Formik uses controlled components + context under the hood. 

Updates to the context's state will trigger a rerender for the entire form. So if you have a large form, performance will suffer.

You can of course bring in memoization to address this, but it's a bit too much in my opinion. Personally I would lean towards a solution that uses uncontrolled components.

I don't feel that it will be a problem here given the small size of the forms here. Absolutely not suggestion to make any changes to this. But just wanted to leave a comment here.